docs/make.jl

```diff
@@ -170,6 +170,6 @@ deploydocs(
           repo = "github.com/CliMA/OceananigansDocumentation.git",
       versions = ["stable" => "v^", "v#.#.#", "dev" => "dev"],
      forcepush = true,
-  push_preview = false,
+  push_preview = true,
```

Comment:
```suggestion
  push_preview = false,
```

Reply:
let's build the docs locally please -- otherwise somebody (and it's always me) needs to clone the docs repo and clean it up, make a new orphan branch and push back again because they get so big in size that GitHub errors and website is not deployed.

Reply:
Sounds good. Sorry. I set this to `true` just to check if all the examples looked alright since building them locally takes quite a long time on my laptop, and forgot to change back.